content/en/docs/user-stories/user-workflow.md

```diff
@@ -6,17 +6,17 @@ weight: 15
 
 Upon logging in, the analyst is greeted by the Dashboard which prominently displays the most significant tags, providing a quick glimpse into the trending topics and themes. This view sets the context for the analyst's tasks ahead.
 
-Next, the analyst navigates to the Analyze tab to initiate a new report, carefully setting its metadata to ensure proper categorization and tracking. This setup is crucial for the coherence and retrieval of the report later in the workflow.
+# Assess
+For easier filtering, in the user setting it is possible to set the end time of a shift.
+To assess your items, go to the Assess section and set the time filter. Apply the "not read" and "not important" filters. Mark as read any items that have been read but considered as unimportant , then work 
+through these items until all are marked as read, only checking summaries. Next, set the filter to "not read" and "important". Verify the importance of each item and mark it 
+as read if not important. Check if the item is in an older report and determine if it adds more value; if not, remove "important" and mark as read. Merge items discussing 
+the same topic. Update the summary manually or use the "summarize with AI" button. Send important or sector-specific items via email. Optionally, up/down vote items and add 
+them to the appropriate reports. Finally, remove the "important" mark and mark as read, or consider using the time filter for better efficiency.
 
-The workflow then transitions to the Assess tab, where the analyst delves into the review process of the latest news stories. Here, the analyst can filter stories either by specific search terms or within a designated time period, utilizing start and end dates to focus on recently gathered news. The interface offers pagination to manage the flow of information effectively, or an infinite scroll for a continuous review, depending on the set parameters.
-
-During the review process in the Assess tab, the analyst has several tools at their disposal to optimize their workflow:
-
-Voting on Stories: The analyst can vote stories up or down, and mark them as important using buttons or keyboard shortcuts (Ctrl+I), to highlight stories of particular relevance.
-Mark as Read: To avoid redundancy and ensure efficiency, the analyst can mark stories as read (or unread if needed) with a button or a shortcut (Ctrl+Space).
-Tags: By applying tag filters, the analyst can narrow down the selection to the most pertinent articles, enhancing focus and efficiency.
-If a story needs editing or creation, the analyst can do so directly within the Assess tab, where each news item can be adjusted until it is locked into a report. This capability is vital for maintaining the freshness and accuracy of the content.
-
-Once the relevant stories are collected and reviewed, the analyst returns to the Analyze tab to finalize the report, marking it as completed. This marks the transition from collection and review to dissemination.
```

Comment:
This section is interesting, maybe we could keep something from it.

Reply:
Yeah maybe just under a:

### additional tasks during assess

subsection?

Reply:
sounds good